qemu/tests/numa_stress.py

```diff
@@ -87,6 +87,8 @@ def run(test, params, env):
     most_used_node, memory_used = max_mem_map_node(host_numa_node, qemu_pid)
 
     for test_round in range(test_count):
+        if os.path.isfile(memory_file):
```

Comment:
Hi @huangyum 
    In my test, host has 4 nodes, and case failed in round 0. So os.path.isfile(memory_file) is False.
    Now, i think the comment 4 in the bug is not a bug.
    If the case turn ERROR as " Don't have enough memory to execute this test after 0 round", it should be the test environment problem.
    What do u think?